packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -0,0 +1,155 @@
+import classNames from 'classnames';
+import * as VaporSVG from 'coveo-styleguide';
+import * as React from 'react';
+import {Input, Svg} from 'react-vapor';
+
+export interface LimitProps {
+    title: string;
+    limitLabel?: string;
+    usageNumber?: number;
+    limitNumber?: number;
```

Comment:
As those 2 props are typed as `number` I don't think its necessary to repeat this information in the props name.
```suggestion
    usage?: number;
    limit?: number;
```